assets/app/scripts/directives/nav.js

```diff
@@ -21,6 +21,10 @@ angular.module('openshiftConsole')
     };
   })
   .directive('projectNav', function($timeout, $location, LabelFilter) {
+    function displayName(project) {
```

Comment:
I think you can use the existing `displayName` filter by adding `$filter` to the function args and calling `$filter('displayName')(project)`